controller/state/record_update.go

```diff
@@ -0,0 +1,10 @@
+package state
+
+type record_update_status int
+
+const (
+	LATEST_UPDATE     record_update_status = 1
+	PREVIOUS_UPDATE   record_update_status = 2
+	UNATTACHED_RECORD record_update_status = 10
+	ATTACHED_RECORD   record_update_status = 11
```

Comment:
interesting choice of casing, for Go :)

Reply:
thoughts on preferred case for these constants? jut `LatestUpdate`?